src/lib/openapi/spec/environment-create-schema.ts

```diff
@@ -0,0 +1,27 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const environmentCreateSchema = {
+    $id: '#/components/schemas/environmentCreateSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['name', 'type'],
+    properties: {
```

Comment:
Could you add a description of what all the properties are and what the environmentCreateSchema itself is? This will replace our current API docs, so it should have all the info developers might need.